packages/grafana-ui/src/components/Gauge/Gauge.tsx

```diff
@@ -110,9 +110,11 @@ export class Gauge extends PureComponent<Props> {
     const valueWidthBase = Math.min(width, dimension * 1.3) * 0.9;
     // remove gauge & marker width (on left and right side)
     // and 10px is some padding that flot adds to the outer canvas
-    const valueWidth = valueWidthBase - ((gaugeWidth + (showThresholdMarkers ? thresholdMarkersWidth : 0)) * 2 + 10);
+    const valueWidth =
+      valueWidthBase -
+      ((gaugeWidth + (showThresholdMarkers ? thresholdMarkersWidth : 0) + (showThresholdLabels ? 10 : 0)) * 2 + 10);
     const fontSize = this.props.text?.valueSize ?? calculateFontSize(text, valueWidth, dimension, 1, gaugeWidth * 1.7);
-    const thresholdLabelFontSize = fontSize / 2.5;
+    const thresholdLabelFontSize = Math.max(fontSize / 2.5, 12);
```

Comment:
unrelated?

Reply:
yea, noticed a Gauge with threshold markers had too big a value, not taking threshold markers into account and noticed threshold markers can become too small to be readable